ssh/DOCS.md

```diff
@@ -1,55 +1,63 @@
-# Home Assistant Add-on: SSH server
+# Home Assistant Add-on: SSH Server
 
 ## Installation
 
 Follow these steps to get the add-on installed on your system:
 
 1. This addon is only visible to "Advanced Mode" users. To enable advanced mode, go to **Profile** -> and turn on **Advanced Mode**.
 2. Navigate in your Home Assistant frontend to **Supervisor** -> **Add-on Store**.
-3. Find the "SSH server" add-on and click it.
+3. Find the "Terminal & SSH" add-on and click it.
 4. Click on the "INSTALL" button.
 
 ## How to use
 
-You need enable the port for external access. You can just enter 22 as value or any other
-value like you want. This recommend to add login credentials.
+This add-on enables two main features to your Home Assistant installation:
+- a Terminal that you can use from your browser, and
+- enable connecting to your system using an SSH client.
 
-To use this add-on, you must have a private/public key to log in.
-To generate them, follow the [instructions for Windows][keygen-windows]
-and [these for other platforms][keygen]. It is possible to set a password for
-login since version 2.0 but for high security use private/public keys.
+Regardless of how you connect (using the Terminal, or a using an SSH client), you end up in this add-on's container. The Home Assistant configuration 
+directory is mounted on the path `/config`.
 
-You can not run both variants at the same time. Enabling login via keys, will
-disable password login.
-
-1. Add a ssh key to  `authorized_keys` or set a `password` in the add-on configuration.
-2. Start the add-on.
-3. Connect to your device using your preferred SSH client and use `root` as
-   the username.
-
-After logging in, you will find yourself in this add-on’s container.
-The Home Assistant configuration directory is mounted on the path `/config`.
-
-Try it out using:
+This add-on comes bundled with [The Home Assistant CLI](https://www.home-assistant.io/hassio/commandline/). Try it out using:
 
 ```bash
 ha help
 ```
 
+### The In-Browser Terminal
+
+To enable the in-browser Terminal all you need to do is start the add-on, and click the Terminal menu item.
```

Comment:
This menu item does not appear by default. It has to be specifically placed into the menu by the end-user. Alternatively, the button to open the web UI on the add-on page can be used.

Reply:
Oh, this is a surprise to me - I don't remember placing it in the sidebar. Where is this setting located?
I suggest something like:
> To access the in-browser Terminal, start the add-on and then click the Open web UI button. If you find yourself using the Terminal a lot, you can add a menu item that takes you directly to the Terminal, by doing X......

Reply:
There is a toggle on the add-on page to toggle the sidebar menu item... It is not enabled by default (this is the same for all add-ons supporting Ingress)

Reply:
Updated.